### PR TITLE
capture joinedSharedSession error and jpg -> jpeg

### PR DESCRIPTION
**Pull request by treykris**
Description
-----------
- Let Sentry capture an error when joinedSharedSession throws one
- Changed invalid accepted file type from `image/jpg` to `image/jpeg`

Developer self-review checklist
-------------------------------
- [x] Potentially confusing code has been explained with comments
- [x] No warnings or errors have been introduced; all known error cases have been handled
- [x] Any appropriate documentation (within the code, README.md, docs, etc) has been updated
- [x] All edge cases have been addressed

### CODE REVIEW

File: src/views/SessionView/Whiteboard.vue [hunk 1]
```diff
@@ -142,7 +142,7 @@
         <input
           type="file"
           class="upload-photo"
-          accept="image/jpg, image/png"
+          accept="image/jpeg, image/png"
```
Comment (jkeat) [inline src/views/SessionView/Whiteboard.vue:145]: Oh good catch! While we're at it, what about using `accept="image/*"`? Could you see if uploading other image types will work ok with zwibbler?

**Review by jkeat — CHANGES_REQUESTED** [review]

File: src/views/SessionView/Whiteboard.vue (hunk 1 above)
Comment (treykris) [reply src/views/SessionView/Whiteboard.vue:145]: Just tried using `accept=image/*`:

- Was able to upload an SVG!
- Uploaded a GIF, but only the first frame is shown, so it's basically converted to an image

Comment (jkeat) [reply src/views/SessionView/Whiteboard.vue:145]: sweet! let's go with that then

**Review by jkeat — APPROVED** [review]
Looks great!

**Pull request merged**